README.md

```diff
@@ -1,86 +1,100 @@
+### Configure FusionAuth
 
-### Configure FusionAuth for load tests
- 
-All of this is optional and will depend upon what you want your tests to do. However, to use the provided examples here, start with these setup steps. 
+#### Create an API Key.
 
-1. Setup an API key. 
-   - If you're a FusionAuth developer, run `sb config` which will insert an API key into the database. 
-   - If you're just an everyday average FusionAuth enthusiast, add an API key however you know how. The examples will expect an API key of `bf69486b-4733-4470-a592-f1bfce7af580` but this can be modified to your liking. 
-   - The key must be a super user key or have permissions to add users and applications as well as log users in. It must work for all tenants.
-2. Set up a tenant. The examples will expect a tenant id of `efb21cfc-fa60-46f4-9598-889151e58517` but that can be modified if you want.
-3. Setup an Application
-   - If needed, update `./src/main/script/setupApplication.sh`. Change the name of the host where FusionAuth runs, if not running it locally.
-   - Run `./src/main/script/setupApplication.sh`  
+1. Go to `https://[your.fusionauth.url]/admin/api-key`.
+1. Click the green + button to create a new API Key.
+1. On the **Add API Key** screen, don't modify or input any values, just click the Save icon.
+1. You should now see your API key. Click on the Key name to reveal the key.
 
-You may also want to change the `factor` in the `User-Registrations.json` file. With no changes, it is `1` which is not very realistic. For a typical FusionAuth deployment, the default is `24000`.
+Notes:
+* If you don't want to create a Super User key, make sure the key at least has permissions to add applications, add users, and log in users, for all tenants.
+* If you're a FusionAuth developer, you can run `sb config` to insert an API key into the database.
 
-### Configuring Load Tests For a Remote FusionAuth Instance
+#### Run the setup script
 
-The load tests are all configured to run against `local.fusionauth.io`. You can run load tests against a FusionAuth instance running at a different hostname, you just need to update the `url` key of each load test JSON file.
+Run the `setup.sh` script to create the FusionAuth Application required for the load tests.
 
+All parameters are optional and will default to values used by FusionAuth's internal development team. If you're not on our development team (and maybe even if you are, if you're not running FusionAuth locally, or not using default values) you'll have to pass in at least `--url` and `--key`:
 ```
-      "url": "https://local.fusionauth.io",
+./setup.sh --url https://[your.fusionauth.url] --key [your-api-key]
 ```
 
-### Run a load test
+⚠️ If your `Default` tenant ID is different from what the script expects, you may get a `400` response code when running the script. In this case, get the ID from your `Default` tenant and pass that in with the `--tenant` flag.
```

Comment:
Not to be too pedantic.. 😎 but all FusionAuth documentation has standardized on `Id` instead of `id` or `ID`.

Reply:
Got it, I'll update.